README.md

```diff
@@ -0,0 +1,194 @@
+<a name="readme-top"></a>
+# 📖 [Demography](#Demography)
+
+📗 Table of Contents
+
+📖 [About the Project](#about-the-project)
+
+🛠 [Built With](#built-with)
+
+- [Tech Stack](#tech-stack)
+- [Key Features](#key-features)
+
+🚀 [Live Demo](#live-demo)
+
+💻 [Getting Started](#getting-started)
+
+- [Setup](#setup)
+- [Prerequisites](#prerequisites)
+- [Install](#install)
+- [Usage](#usage)
+- [Run tests](#run-test)
+- [Deployment](#deployment)
+
+👥 [Authors](#authors)
+
+🔭 [Future Features](#future-features)
+
+🤝 [Contributing](#contributing)
+
+⭐️ [Show your support](#show-your-support)
+
+🙏 [Acknowledgments](#acknowledgment)
+
+❓ [FAQ (OPTIONAL)](#faq)
+
+📝 [Credit](#credit)
+
+📝 [License](#licence)
+
+<br><br>
+
+## 📖 [Demography](#Demography)
+### 📖 <a name="about-the-project">Description </a>
+
+The [Demography project](https://github.com/kambereBr/demography) is a web application that fetches data from <b>countriesnow.space</b> API and list demographic data (population) for all countries or states all over the world from 1960 to 2018. Users can display the list of demographic data, filter data by the range of alphabetic letters and display details for desired country or state. The project uses reducers and actions, Redux in React components and makes API calls to external APIs.
+
+[`Back to top`](#readme-top)
+
+### 🛠 <a name="project-built-with">Built With </a>
+The following technologies have been used in developping this project. 
+
+#### <a name="tech-stack"> Tech Stack
+</a>
+
+```[Tech-stack]
+client: React, Redux, React-Redux, Redux Toolkit, Jest, HTML, CSS and Javascript.
+````
+
+[`Back to top`](#readme-top)
+
+### <a name="key-features"> Key Features </a>
+
+`Demography` key features are the following main projects:
+
+```
+Project-feature1: Data Fetching from the external API 
+Project-feature2: Rendering of all demographic data for all countries
+Project-feature3: Countries filtering
+Project-feature4: Country details
+```
+
+[`Back to top`](#readme-top)
+
+## 🚀 <a name="live-demo"> Live Demo </a>
+
+You can access the Demograpy app at the link below.
+
+- [Live Demo Link](https://kamberebr.github.io/demography)
```

Comment:
- [ ] Please kindly make sure your deployed link works correctly and to fix it you need to delete in your package.json file line two that precise the home page to be `https://kamberebr.github.io/demography` because even when we run the app locally the data doesn’t render, it's blank page. After fixing that deploy again your project🙃 

<img width="1110" alt="Screenshot 2023-09-09 at 00 41 10" src="https://github.com/kambereBr/demography/assets/74501523/ded72b6a-0c27-447b-bca5-2e4692af393b">